data/schema.rdfa

```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastEvent">BroadcastEvent</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Boolean">Boolean</a></span>
+    </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/CableOrSatelliteService">
+      <span class="h" property="rdfs:label">CableOrSatelliteService</span>
+      <span property="rdfs:comment">A service which provides access to media programming like TV or radio. Access may be via cable or satellite.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Organization">Organization</a></span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Service">Service</a></span>
+   </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/BroadcastChannel">
+      <span class="h" property="rdfs:label">BroadcastChannel</span>
+      <span property="rdfs:comment">A unique instance of a BroadcastService on a CableOrSatelliteService lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Intangible">Intangible</a></span>
+   </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/TelevisionChannel">
+      <span class="h" property="rdfs:label">TelevisionChannel</span>
+      <span property="rdfs:comment">A unique instance of a television BroadcastService on a CableOrSatelliteService lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/BroadcastChannel">Intangible</a></span>
+   </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/RadioChannel">
+      <span class="h" property="rdfs:label">RadioChannel</span>
+      <span property="rdfs:comment">A unique instance of a radio BroadcastService on a CableOrSatelliteService lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/BroadcastChannel">Intangible</a></span>
+   </div>
+    <div typeof="rdf:Property" resource="http://schema.org/broadcastChannelId">
+      <span class="h" property="rdfs:label">broadcastChannelId</span>
+      <span property="rdfs:comment">The unique address by which the BroadcastService can be identified in a provider lineup. In US, this is typically a number.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastChannel">BroadcastChannel</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/inBroadcastLineup">
+      <span class="h" property="rdfs:label">inBroadcastLineup</span>
+      <span property="rdfs:comment">The CableOrSatelliteService offering the channel.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastService">BroadcastService</a></span>
```

Comment:
Should the domain be BroadcastService or BroadcastChannel? The examples have it on BroadcastChannel, which makes more sense to me.